src/components/layout/header/styles.ts

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components'
+
+export const Container = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 100px;
+`
+
+export const Logo = styled.div`
+  background-image: url('assets/logo.svg');
+  width: 182px;
+  height: 36px;
+  @media (max-width: ${(props) => props.theme.breakpoints.lg}) {
```

Comment:
Sice to zatím nemáme v CONTRIBUTING, ale tady navrhuji rozložit props v rámci funkce: `({ theme }) => theme...`. Šlo by předělat na všech místech?